infrastructure/terraform/madoc/single-instance/files/backup/backup-mysql.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/bash
+/usr/bin/mysqldump --single-transaction --skip-lock-tables -h 0.0.0.0 -P $MYSQL_PORT -u $MYSQL_USER $MYSQL_DATABASE --password=$MYSQL_PASSWORD > $MYSQL_DATABASE.sql
```

Comment:
I think more people recognize `127.0.0.1` as loopback vs `0.0.0.0`.

Reply:
Agreed - not sure why I picked that!?